package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "puppeteer-utils",
```

Comment:
NPM recommends prefixing with organization name these days — `@automattic/puppeteer-utils`

Reply:
Thanks! updated that